Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift

```diff
@@ -29,12 +48,14 @@ public struct ClientConfiguration {
     
     public init(environment: Environment, clientId: String, redirectURI: URL) {
         self.init(serverURL: URL(string: environment.rawValue)!,
+                  sessionServiceURL: URL(string: environment.sessionService)!,
                   clientId: clientId,
                   redirectURI: redirectURI)
     }
     
-    public init(serverURL: URL, clientId: String, redirectURI: URL) {
+    init(serverURL: URL, sessionServiceURL: URL, clientId: String, redirectURI: URL) {
```

Comment:
This one probably could be unpublic @zamorastp

Reply:
Are there some scenarios where they would like to use this method?  Test against another server?

Reply:
Perhaps. Making public again

Reply:
If we dont know, lets go and make it private. Someone will ask if they have used it ;)

Reply:
Back to internal then ;). This will be a breaking API change then. FYI